Library/Homebrew/keg_relocate.rb

```diff
@@ -1,4 +1,12 @@
+require "os/mac/relocate"
```

Comment:
Yeah, sounds good to rename `os/mac/relocate.rb` to `os/mac/keg.rb`. The whole idea is trying to keep os related codes inside `os` folder.